test/e2e/projects_test.go

```diff
@@ -68,7 +68,7 @@ var _ = Describe("Nutanix projects", Label("capx-feature-test", "projects"), fun
 			invalidProjectNMT := testHelper.createDefaultNMT(clusterName, namespace.Name)
 			invalidProjectNMT.Spec.Template.Spec.Project = &infrav1.NutanixResourceIdentifier{
 				Type: "name",
-				Name: pointer.StringPtr(nonExistingProjectName),
+				Name: pointer.String(nonExistingProjectName),
```

Comment:
functions from the pointer library got renamed.

Reply:
Get ready to change these to `ptr` soon, the `pointer` library is deprecated in k8s/k8s 1.29

Reply:
Thanks for the insight, updated to ptr.